composables/viewer.ts

```diff
@@ -6,3 +6,10 @@ export type ViewerProject = {
 export type ViewerProjectList = {
   items: ViewerProject[];
 };
+
+export enum ViewObject {
+  Preview,
+  Editor, // Future use case
+  AlwaysEnabled,
+  AlwaysDisabled,
+}
```

Comment:
Maybe name it `ViewContext` with `Viewer`, `Backpack`, and `Editor` states.

Reply:
`ViewContext` is fine, `backpack` is doable, but def need a disabled state.